options/locale/locale_en-US.ini

```diff
@@ -138,6 +138,11 @@ ssl_mode = SSL
 charset = Charset
 path = Path
 sqlite_helper = File path for the SQLite3 database.<br>Enter an absolute path if you run Gitea as a service.
+reinstall_error = You are trying to re-install in an existing Gitea database
+reinstall_confirm_message = In most cases, you should use your existing "app.ini" to run Gitea. Re-installing in an existing Gitea database would cause potential problems. If you know what you are doing, mark these tips as checked to continue.
+reinstall_confirm_check_1 = The data encrypted by the secret keys in app.ini will be lost, users may not be able to log in with 2FA/OTP.
```

Comment:
I will check the secrets in backend code, will update later.

Reply:
The code in `install.go` always creates new secrets. I think we shouldn't make the logic too complex (checking everything and fixing every user mistake, it would cause more bugs)

If a full `app.ini` only loses `INSTALL_LOCK`, it must be a very rare case, we can just add a message here to tell user to fix/recover the correct `app.ini`. Anyway, we can help users, but we can not do everything for users.

Reply:
After some thought, I think we can use this logic:

If there was a secret key in app.ini, we do not overwrite it, then there will be no problem.